src/profile-logic/process-profile.js

```diff
@@ -813,6 +813,35 @@ function _adjustMarkerTimestamps(
           newData.timeStamp += delta;
         }
       }
+      if (newData.type === 'Network') {
```

Comment:
Can you mention that this code is duplicated in the upgrader? I think that will help if we need to touch this code in the future, to know that it's duplicated elsewhere. I think the decision to duplicate is good, but it would be nice to call it out.

Reply:
Done. I didn't add a comment specifically to the network piece; it's going to become irrelevant and distracting soon. Instead I added a more generic comment at the end of the function.